srcpkgs/paraview/template

```diff
@@ -0,0 +1,64 @@
+# Template file for 'paraview'
+pkgname=paraview
+version=5.6.1
+revision=1
+archs="x86_64"
+wrksrc=ParaView-v${version}
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release
+ -DCMAKE_INSTALL_PREFIX=/usr
+ -DPARAVIEW_ENABLE_FFMPEG=ON
+ -DPARAVIEW_ENABLE_GDAL=ON
+ -DPARAVIEW_ENABLE_MATPLOTLIB=ON
+ -DPARAVIEW_ENABLE_PYTHON=ON
+ -DPARAVIEW_USE_MPI=ON
+ -DPARAVIEW_USE_VISITBRIDGE=OFF
+ -DPARAVIEW_USE_OSPRAY=OFF
+ -DVTK_PYTHON_FULL_THREADSAFE=ON
+ -DVTK_PYTHON_VERSION=3
+ -DVTK_SMP_IMPLEMENTATION_TYPE=TBB
+ -DVTKm_ENABLE_MPI=ON
+ -DVTKm_ENABLE_RENDERING=ON
+ -DVTKm_USE_DOUBLE_PRECISION=ON
+ -DVTK_USE_SYSTEM_LIBRARIES=OFF
+ -DVTK_USE_SYSTEM_DOUBLECONVERSION=ON
+ -DVTK_USE_SYSTEM_EIGEN=ON
+ -DVTK_USE_SYSTEM_EXPAT=ON
+ -DVTK_USE_SYSTEM_FREETYPE=ON
+ -DVTK_USE_SYSTEM_GLEW=ON
+ -DVTK_USE_SYSTEM_HDF5=ON
+ -DVTK_USE_SYSTEM_JPEG=ON
+ -DVTK_USE_SYSTEM_JSONCPP=ON
+ -DVTK_USE_SYSTEM_LIBPROJ=ON
+ -DVTK_USE_SYSTEM_LIBXML2=ON
+ -DVTK_USE_SYSTEM_LZMA=ON
+ -DVTK_USE_SYSTEM_MPI4PY=ON
+ -DVTK_USE_SYSTEM_NETCDF=ON
+ -DVTK_USE_SYSTEM_OGG=ON
+ -DVTK_USE_SYSTEM_PEGTL=OFF
+ -DVTK_USE_SYSTEM_PNG=ON
+ -DVTK_USE_SYSTEM_PUGIXML=ON
+ -DVTK_USE_SYSTEM_THEORA=ON
+ -DVTK_USE_SYSTEM_TIFF=ON
+ -DVTK_USE_SYSTEM_ZLIB=ON
+ -DVTK_USE_SYSTEM_PROTOBUF=ON"
+hostmakedepends="cmake"
```

Comment:
No need to add `cmake` to `hostmakedepends` if the `build_style` is set to `cmake`

Reply:
`hostmakedepends="cmake"` removed in 3c065ef682